package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "bytes",
+  "name": "@daveobriencouk/bytes",
```

Comment:
I think you changed the name here accidentally; I can't accept the PR if it'll rename the module :) Just revert this from the PR 👍